### ISSUE TITLE
Feature Request: Change in accent color for light mode

### ISSUE THREAD

**Issue by metronightly** [Sandakan/Nora#153]
[Image](https://user-images.githubusercontent.com/75716685/228143373-e3d88752-ef89-4dfb-a455-a6f3229072c7.jpg)

Currently the color on the left is the default accent color for both light and dark theme. My suggestion is using the color on the right as accent color for light theme and the color on the left for dark mode only. This way the app would look better in my opinion.

**Comment by Sandakan** [comment]
I don't think that would be pleasing to the eye. 

![image](https://user-images.githubusercontent.com/45460443/228218880-e16a304f-f795-46e3-abb9-2faea373b2dc.png)

**Comment by metronightly** [comment]
> I don't think that would be pleasing to the eye.
> 
> ![image](https://user-images.githubusercontent.com/45460443/228218880-e16a304f-f795-46e3-abb9-2faea373b2dc.png)


Yeah this is a bit too dark 😨 Have you tried with some lighter shades than this ? Something in the middle between this and the current default nora color.

**Comment by metronightly** [comment]
> I don't think that would be pleasing to the eye.
> 
> ![image](https://user-images.githubusercontent.com/45460443/228218880-e16a304f-f795-46e3-abb9-2faea373b2dc.png)

Okay I went back to oto and I just found why it looked bad on Nora while the same color looks good on Oto.

So on Oto while we select an option the text color changes from black to the right side blue and the background for that option is the left side blue. 
![Like This](https://user-images.githubusercontent.com/75716685/228221790-0bf0cd8e-4d99-4736-a327-782cbb12ff6a.jpg)

On the screenshot you provided the background is the right color blue while the text is in black.

Ignore my previous reply.

**Comment by Sandakan** [comment]
I still don't think that's a good color for the sidebar. The color seems odd with the background color in light blue. If the background were white, it would've been better.

![image](https://user-images.githubusercontent.com/45460443/228224313-5cacfb69-3e36-4760-be88-6e425d3be30f.png)

**Comment by metronightly** [comment]
> I still don't think that's a good color for the sidebar. The color seems odd with the background color in light blue. If the background were white, it would've been better.
> 
> ![image](https://user-images.githubusercontent.com/45460443/228224313-5cacfb69-3e36-4760-be88-6e425d3be30f.png)

Hm honestly this looks a lot better to me, but it's entire up to you of course.

Having said that, have you tried changing the color of the headings inside settings to this blue? 
![Image](https://user-images.githubusercontent.com/75716685/228227284-cdfcde8f-7744-4dbd-b3db-59c0fedf4d7f.png)

I think you'll like that 😅

**Comment by Sandakan** [comment]
Also, there are some contrast issues when using that blue color in the sidebar.

![image](https://user-images.githubusercontent.com/45460443/228233383-9d1f721a-0285-43da-9cc2-d78eb9740568.png)

Regarding the title color of the Settings, since I'm using a palette of colors throughout the app, changing the title color only in the Settings won't be possible because the changed color could look in one place but won't be in another place.

Sorry to disappoint you with this.

**Comment by metronightly** [comment]
> Also, there are some contrast issues when using that blue color in the sidebar.
> 
> ![image](https://user-images.githubusercontent.com/45460443/228233383-9d1f721a-0285-43da-9cc2-d78eb9740568.png)
> 
> Regarding the title color of the Settings, since I'm using a palette of colors throughout the app, changing the title color only in the Settings won't be possible because the changed color could look in one place but won't be in another place.
> 
> Sorry to disappoint you with this.

No worries at all.

**Comment by metronightly** [comment]
Since you haven't closed this yet, I'm guessing you're open to adding custom accent colors sometime in the future ? 😁

**Comment by Sandakan** [comment]
The current CSS framework I'm using prevents me from adding custom accent colors. Since it provides many advantages over traditional CSS, removing this framework just to add custom accent colors won't be a good solution. 😥
